core/tests/Test.Microsoft.Identity.Core.Unit/Mocks/MockHttpMessageHandler.cs

```diff
@@ -47,6 +47,8 @@ internal class MockHttpMessageHandler : HttpMessageHandler
 
         public IDictionary<string, string> PostData { get; set; }
 
+        public IDictionary<string, object> PostDataObject { get; set; }
```

Comment:
Shouldn't this be used  in this class?

Reply:
this is needed for the mocks, to handle the securestring.

---
In reply to: [219473186](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1261#discussion_r219473186) [](ancestors = 219473186)